util/microservice-sdk/js/examples/back-pressure/sample.js

```diff
@@ -1,5 +1,6 @@
 const { logger } = require('@dojot/dojot-module-logger');
-const ConsumerBackPressure = require('../lib/kafka/ConsumerBackPressure');
+const { Kafka: { ConsumerBackPressure } } = require('../index.js');
```

Comment:
I don't like the name `ConsumerBackPressure` is a little weird. Maybe something like BPConsumer would be better, but this is not the goal of this PR.